README.md

```diff
@@ -1,6 +1,106 @@
+# Logstash Filter Aggregate Documentation
```

Comment:
Thanks for appreciation :)

For the sample place (there is only one), my idea was to explain the plugin utility by an example.
A lot of developers are more comfortable with a nice example than a long description.
And in the next section, I go deeply, with core explanation : "how it works".
And finally, I go deeply with a "options reference" section.
But if you think this can be better organized, it's possible :)